radio/src/gui/128x64/lcd.cpp

```diff
@@ -588,29 +620,18 @@ void lcdDrawNumber(coord_t x, coord_t y, lcdint_t val, LcdFlags flags, uint8_t l
         lcdDrawChar(x, y, '.', f);
       }
     }
-    if (dblsize && (lcduint_t)val >= 1000 && (lcduint_t)val < 10000) x-=2;
+    if (dblsize && (lcduint_t)val >= 1000 && (lcduint_t)val < 10000) {
+      x-=2;
+    }
     val = qr.quot;
     x -= fw;
 #if defined(BOLD_FONT) && !defined(CPUM64) || defined(TELEMETRY_NONE)
-    if (i==len && (flags & BOLD)) x += 1;
-#endif
-  }
-
-  if (xn) {
-    if (midsize) {
-      if ((flags&INVERS) && ((~flags & BLINK) || BLINK_ON_PHASE)) {
-        lcdDrawSolidVerticalLine(xn, y, 12);
-        lcdDrawSolidVerticalLine(xn+1, y, 12);
-      }
-      lcdDrawSolidHorizontalLine(xn, y+9, 2);
-      lcdDrawSolidHorizontalLine(xn, y+10, 2);
-    }
-    else {
-      // TODO needed on CPUAVR? y &= ~0x07;
-      lcdDrawSolidFilledRect(xn, y+2*FH-3, ln, 2);
```

Comment:
Are you sure this `lcdDrawSolidFilledRect` not needed anymore for PREC1 / PREC2 with DBLSIZE fonts?